modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -21,6 +21,12 @@ definition {
 			EVPUtils.addEVPSite();
```

Comment:
Hey @juliosilveira77, none of these changes modified the LPS-186802 as mentioned in the commit title.
could you please set a appropriate commit title?